### ISSUE TITLE
Allow adverbs & adjectives

### ISSUE THREAD

**Issue by pigsonthewing** [googleknowledge/qlabel#2]
The food menu example, http://googleknowledge.github.io/qlabel/demo/menu/, uses the adjective "vegetarian", but when translated, that becomes the noun "vegetarianism". qLabel should allow the use of adjectives, and adverbs, perhaps via a switch in the HTML markup, and relying on the forthcoming integration between Wikidata and Wiktionary.

**Comment by brightbyte** [comment]
Doing this would require Wikidata to function as a dictionary. There is a proposal for this, but we are no where near this functionality: https://www.wikidata.org/wiki/Wikidata:Wiktionary

Right now, Wikidata can be used as a thesaurus: it gives the you names of things in different languages. That's excellent for labeling maps, for tagging, etc. 

Wikidata is not ready to be used for translations or internationalization. That will have to wait until we have proper support for modelling lexical entities.

**Comment by pigsonthewing** [comment]
@brightbyte Hence: "relying on the forthcoming integration between Wikidata and Wiktionary"!

**Comment by vrandezo** [comment]
I fully agree with this feature request, and I unfortunately expect that it will stay open for quite a while.
